<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="search_param_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметр поиска:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="search_param_box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Автор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Название</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="search_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="view_param_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вид отображения:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="view_param_box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Список</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Таблица</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stacked_widget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>601</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="list_page">
     <widget class="QListWidget" name="book_info_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="table_page">
     <widget class="QTableWidget" name="book_info_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
